tests/test_traversal.py

```diff
@@ -0,0 +1,189 @@
+import numpy as np
+import tskit
+
+import phylokit as pk
+
+
+class TestBalancedBinaryOdd:
+    # 2.00┊   4   ┊
+    #     ┊ ┏━┻┓  ┊
+    # 1.00┊ ┃  3  ┊
+    #     ┊ ┃ ┏┻┓ ┊
+    # 0.00┊ 0 1 2 ┊
+    #     0      1
+
+    def tsk_tree(self):
+        return tskit.Tree.generate_balanced(3)
+
+    def tree(self):
+        return pk.from_tskit(tskit.Tree.generate_balanced(3))
+
+    def test_postorder(self):
+        assert np.array_equal(pk.postorder(self.tree()), self.tsk_tree().postorder())
+
+    def test_preorder(self):
+        assert np.array_equal(pk.preorder(self.tree()), self.tsk_tree().preorder())
+
+    def test_postorder_from_node(self):
+        assert np.array_equal(pk.postorder(self.tree(), 3), np.array([1, 2, 3]))
+
+    def test_preorder_from_node(self):
+        assert np.array_equal(pk.preorder(self.tree(), 3), np.array([3, 1, 2]))
+
+
+class TestBalancedBinaryEven:
+    # 2.00┊    6    ┊
+    #     ┊  ┏━┻━┓  ┊
+    # 1.00┊  4   5  ┊
+    #     ┊ ┏┻┓ ┏┻┓ ┊
+    # 0.00┊ 0 1 2 3 ┊
+    #     0         1
+    def tsk_tree(self):
+        return tskit.Tree.generate_balanced(4)
+
+    def tree(self):
+        return pk.from_tskit(tskit.Tree.generate_balanced(4))
+
+    def test_postorder(self):
```

Comment:
Let's just put the actual nodes in here to the test and forget about being exactly the same as ``tskit``. So, we'd have
```
assert np.array_equal(pk.postorder(self.tree()), [3, 2, 5, 1, 0, 4, 6])
```
Or something equivalent.